modules/ivc_champva/app/models/ivc_champva/vha_10_7959a.rb

```diff
@@ -22,6 +22,7 @@ def metadata
         'source' => 'VA Platform Digital Forms',
         'docType' => @data['form_number'],
         'businessLine' => 'CMP',
+        'ssn_or_tin' => @data.dig('veteran', 'ssn_or_tin'),
```

Comment:
This line will have to be
`'ssn_or_tin' => @data.dig('applicant_member_number'),`